src/test/java/org/broadinstitute/hellbender/utils/codecs/gtf/EnsemblGtfCodecUnitTest.java

```diff
@@ -0,0 +1,174 @@
+package org.broadinstitute.hellbender.utils.codecs.gtf;
+
+import org.broadinstitute.hellbender.GATKBaseTest;
+import org.testng.Assert;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Test class for the ENSEMBL GTF Reader.
+ * Modeled after the TableCodecUnitTest, with extras specific to this file format.
+ * Created by jonn on 2020 02 28.
+ */
+public class EnsemblGtfCodecUnitTest extends GATKBaseTest {
```

Comment:
It would be good to have a test in here that reads the header and decodes some records.  I know there are integration test that do that, but it's good to have one here too that's separate from funcotator.  It doesn't have to be extremely extensive, just enough to show that it returns a few expected records.

Reply:
Sounds good.  I'll make a test for it.